data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_spells.lst

```diff
@@ -86,16 +86,16 @@ Vestigewrack	SOURCEPAGE:p.54	TYPE:Divine.Arcane	SCHOOL:Necromancy	DESCRIPTOR:Evi
-Ronove		KEY:Bind Vestige ~ Ronove		TYPE:Vestige	RANGE:Personal	TARGETAREA:You		DURATION:24 hours	SOURCEPAGE:P.	DESC:The Iron Maiden. Binding DC:15	DESC:&nl;Special - Ronoves seal must be drawn in the soil, under the sky	DESC:&nl;Sign - the flesh of your face forms into a natural smile (or frown if you failed your binding check)	DESC:&nl;Influence - You feel that others doubt your abilities and competence. Despite what anyone says, you feel the constant need to prove your worth. In Addition, Ronove requires that you consume neither food nor beverages (including potions) for the entire time you remain bound to her	DESC:&nl;Abilities - Ronove gives you the power to fall any distance without harm, lift objects without touching them, and run like the wind.	DESC:&nl;Cold iron and magic attacks; your melee attacks are considered magic for overcoming damage reduction|PREVARLTEQ:BinderEffectiveLVL,6	DESC:&nl;Cold iron and magic attacks; your melee attacks are considered magic and cold iron for overcoming damage reduction|PREVARGTEQ:BinderEffectiveLVL,7	DESC:&nl;Far hand; as a swift action you can move unattended objects within %1 feet up to 5' per rnd, but no more than 5' from the ground. This has an effective strength score of %2; Alternatively you can use this to attack, dealing 1d6 damage as a bull rush|10*BinderEffectiveLVL|BinderEffectiveLVL	DESC:&nl;Feather fall; you fall slowly, at a mere 60 feet per round [equivalent to the end of a fall from a few feet], and take no damage upon landing.	DESC:&nl;Ronove's fists; you gain the benefits of the 'Improved Unarmed strike', as a monk, but use your binder level	DESC:&nl;Sprint; gain +10 move speed
+Amon			KEY:Bind Vestige ~ Amon			TYPE:Vestige	RANGE:Personal	TARGETAREA:You		DURATION:24 hours	SOURCEPAGE:P.21	DESC:&nl;The Void Before the Alter. Binding DC:20, special requirement: cannot have bound certain other vestiges within 24 hours.	DESC:&nl;Sign - You grow a ram's curling horns	DESC:&nl;Influence - you become surly and irritable. You must resist even beneficial spells from devotees of Fire,Sun or Law as Amon despises such dieties	DESC:&nl;Darkvision 60'	DESC:&nl;Fire breath - as a standard action, vomit a line of fire %1 feet long doing %2d6 damage to each creature in the line (Reflex save DC:%3 for half damage), can breath once per 5 rnds|10*BinderEffectiveLVL|BinderEffectiveLVL|BinderEffectiveLVL/2+10+CHA|PREVARLTEQ:BinderEffectiveLVL,5	DESC:&nl;Fire breath - as a standard action, vomit a line of fire %1 feet long doing %2d6 damage to each creature in the line (Reflex save DC:%3 for half damage), can breath once per 5 rnds|50|BinderEffectiveLVL|BinderEffectiveLVL/2+10+CHA|PREVARGTEQ:BinderEffectiveLVL,6	DESC:&nl;Ram Attack - Use your Rams horns as a natural weapon to do 1d6+%1 damage, plus an extra 1d8 when charging|1.5*STR
```

Comment:
Mate, I am having a hard time deciphering what is actually changed in this file.

Reply:
The beginging of each DESC tag now contains a oneliner 'catch phrase' or 'flavour text' for the vestige, ie 'Amon' is referred to as 'The Void Before the Alter', and then I also standardised the format to add the binding DC and special requirements in order, before the bulk of the text - some of the vestiges were missing bits of this info, or had it in a different order. Note the '&nl' to force newlines - I've got those in a few DESC tags but not all of them are (yet) available, have JIRA's in to get the PDF to handle them in more places, so they will eventually work.

Reply:
this one is much easier to read in multiline format, as I am using multiple DESC tags - various abilities for these 'spell's are level dependant and have vars substituting in too.